vpr/src/device/rr_graph_view.h

```diff
@@ -248,6 +248,19 @@ class RRGraphView {
     inline const char* node_side_string(RRNodeId node) const {
         return node_storage_.node_side_string(node);
     }
+    /*Get the switch id that represents the iedge'th outgoing edge from a specific node
```

Comment:
Please follow the doxygen compatible format for comments in this file.
It should start with ``/**``
Please refer to other code comments as examples.